### PR TITLE
feat: add world chain + xlayer + cheese chains

### PR DESCRIPTION
**Pull request by avious00**
- Add support for cheeschain, xlayer and worldchain

### CODE REVIEW

**Comment by avious00** [comment]
@Mo-Hussain can you revert the changes for arthera? since they're going to attempt to run themselves they'll use their registry addresses https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/chains/arthera/addresses.yaml

**Comment by jmrossy** [comment]
@avious00 @Mo-Hussain A few things still to fix here (see CI). Lmk if you'd like some help with it.
I also see there's redundancy with PR #106 from @rshtirmer. Should this supersede that or would you like to get that merged first?

File: chains/xlayer/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://www.oklink.com/api
```
Comment (Mo-Hussain) [inline chains/xlayer/metadata.yaml:3]: I've left as is, it does not fall under one of the families @tkporter

**Comment by Mo-Hussain** [comment]
> @avious00 @Mo-Hussain A few things still to fix here (see CI). Lmk if you'd like some help with it. I also see there's redundancy with PR #106 from @rshtirmer. Should this supersede that or would you like to get that merged first?

**Comment by Mo-Hussain** [comment]
> @avious00 @Mo-Hussain A few things still to fix here (see CI). Lmk if you'd like some help with it. I also see there's redundancy with PR #106 from @rshtirmer. Should this supersede that or would you like to get that merged first?

@jmrossy this PR will supersede PR #106, @avious00 is coordinating with cheesechain

**Review by tkporter — APPROVED** [review]

**Review by jmrossy — APPROVED** [review]

**Pull request merged**